docs/intro/tutorial.rst

```diff
@@ -372,10 +407,10 @@ Let's add this code to our spider::
         ]
      
         def parse(self, response):
-            for sel in response.xpath('//ul/li'):
-                title = sel.xpath('a/text()').extract()
+            for sel in response.xpath('//div[@class="title-and-desc"]'):
```

Comment:
The transition from `for sel in response.xpath('//div[@id="site-list-content"]/...` to `for sel in response.xpath('//div[@class="title-and-desc"]'):` is not explained.